package schemas;


option java_package = "com.segmetics.ng.protocol.message";
//type 000000

message RequestProto {
    required string msgId = 1;
    required int32 version = 2;
    required string hostId = 3;
    required int32 type = 4;
    optional string payload = 5;
}

message RequestRespProto {
    required string msgId = 1;
    required int32 version = 2;
    required string hostId = 3;
    required string reqestId = 4;
    required int32 type = 5;
    optional string payload = 6;
}
